### ISSUE TITLE
update terraform files 

### ISSUE THREAD

**Issue by mhost39** [freeflowuniverse/freeflow_caprover#5]
env_vars should be 
```
 env_vars = {
      PUBLIC_KEY = "your ssh"
      SWM_NODE_MODE = "leader"
      CAPROVER_ROOT_DOMAIN = "gridtesting.xyz"
    }
```

and how set network (testnet or devnet) need to export ``` export NETWORK="<network>" # dev or test ```

**Comment by xmonader** [comment]
why would we use ed25519 ?

**Comment by sameh-farouk** [comment]
@mhost39 thanks for pointing this out, I'm closing this as it was fixed in fe2be262de587
